beetmoverscript/script.py

```diff
@@ -120,18 +124,31 @@
                             manifest['mapping'][locale][artifact]['destinations']]
 
             balrog_manifest = manifest['mapping'][locale][artifact].get('update_balrog_manifest')
+            # For partials
+            from_buildid = manifest['mapping'][locale][artifact].get('from_buildid')
             beets.append(
                 asyncio.ensure_future(
                     move_beet(context, source, destinations, locale=locale,
                               update_balrog_manifest=balrog_manifest,
+                              from_buildid=from_buildid,
                               artifact_pretty_name=artifact_pretty_name)
```

Comment:
I know this predates your change, but I find it odd that we're using kwargs in the call, and args in the function. I suppose this gives us readability on the calling side and allows us to require all args on the function side?